src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/Pipe.kt

```diff
@@ -88,7 +87,7 @@ class Pipe<E>(
 
         fun <E> Executor.newPipe(
             name: String,
-            queue: Queue<E> = MpscChunkedArrayQueue(16384),
+            queue: Queue<E> = MpscUnboundedArrayQueue(65536),
```

Comment:
If the queue grows infinitely we can face an OOM error. Maybe implement any backpressure logic and calculate the max volume according to available resources.

Reply:
Events and custom logic are important for us. But what should the box do if any unpredictable happened?

Reply:
> Events and custom logic are important for us. But what should the box do if any unpredictable happened?

I don't get it. Could you please elaborate?

Reply:
I mean about internal back pressure, for example, event router will block the thread because estore's queue is full.

Reply:
It'll block a thread calling event batcher which could lead to executor's queue growing

Reply:
We should think about back pressure implementation based on the handler's protocol in future.
This functionality will be required when the back pressure in common will be implemented.